.github/workflows/brakeman-analysis.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow integrates Brakeman with GitHub's Code Scanning feature
+# Brakeman is a static analysis security vulnerability scanner for Ruby on Rails applications
+
+name: Brakeman Scan
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ main ]
+  schedule:
+    - cron: '29 18 * * 3'
+
+jobs:
+  brakeman-scan:
+    name: Brakeman Scan
+    runs-on: ubuntu-latest
+    steps:
+    # Checkout the repository to the GitHub Actions runner
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    # Customize the ruby version depending on your needs
+    - name: Setup Ruby
```

Comment:
[nit] we have some inconsistency with things like `Setup Ruby` and `Set up Ruby` -- we should probably make it consistent. (I think `Set up Ruby` is correct in terms of grammar but I just want it to be consistent!)

Reply:
Do these basic steps even need names? Or should we just rely on the fact that the action is called `actions/setup-ruby` to document it?

Reply:
i think it's nice that we get the steps in the actual action, so i think i would prefer to keep them, but if that helper thing works out we could bundle them all together under the same step name? maybe?

<img width="361" alt="Screen Shot 2021-08-06 at 4 15 40 PM" src="https://user-images.githubusercontent.com/5473830/128566533-091e6e57-7983-4c53-9684-d0688930f088.png">